lib/transports.js

```diff
@@ -1,14 +1,16 @@
 var events = require('events');
 var util = require('util');
+var assign = require('object-assign');
```

Comment:
It's a simple for loop inside)) https://github.com/sindresorhus/object-assign/blob/master/index.js#L24-L39

I can copy-paste lib code to `lib/utils.js` but are you sure this is better?